world_universities_and_domains.json

```diff
@@ -127639,5 +127645,1158 @@
+    "domains": [
+      "smail.hgnu.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.aust.edu.cn/"
+    ],
+    "name": "Anhui University of Science & Technology",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "aust.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.muc.edu.cn/"
+    ],
+    "name": "Minzu University of China",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "muc.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.haut.edu.cn/"
+    ],
+    "name": "Henan University of Technology",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "stu.haut.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.fmp.edu.cn/"
+    ],
+    "name": "Fuzhou Melbourne Polytechnic Vocational College",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "st.fmp.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.zcst.edu.cn/"
+    ],
+    "name": "Zhuhai College of Science and Technology",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "stu.zcst.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.esiee-it.fr/fr"
+    ],
+    "name": "ESIEE-IT",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "edu.esiee-it.fr"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://www.niser.ac.in/"
+    ],
+    "name": "National Institute of Science Education and Research",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "niser.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://karunya.edu/"
+    ],
+    "name": "Karunya Institute of Technology and Sciences",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "karunya.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.gvpce.ac.in/"
+    ],
+    "name": "Gayatri Vidya Parishad College of Engineering",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "gvpce.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://indoreinstitute.com/"
+    ],
+    "name": "Indore Institute of Science and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "indoreinstitute.com"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://hcst.edu.in/"
+    ],
+    "name": "Hindustan College of Science and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "sgei.org"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "www.andhrauniversity.edu.in"
```

Comment:
Really minor thing, but protocol missing here

Reply:
:) that is the missing one. Thank you!

Reply:
Found out this one is a duplicated university :). Will fix it.

Reply:
You can see the fix in this commit 50e2f3527258f8be170102599a4746c31af0b6a6